src/kOS.Safe.Test/Serialization/TerminalFormatterTest.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using kOS.Safe.Encapsulation;
+using kOS.Safe.Exceptions;
+using NUnit.Framework;
+using kOS.Safe.Serialization;
+
+namespace kOS.Safe.Test.Serialization
+{
+    [TestFixture]
+    public class TerminalFormatterTest
+    {
+        [SetUp]
+        public void Setup()
+        {
+
+        }
+
+        [Test]
+        public void CanSerializeLexicons()
+        {
+            var lex = new Lexicon();
+            var nested = new Lexicon();
+
+            lex["key1"] = "value1";
+            lex["key2"] = 1;
+            lex["key3"] = nested;
+
+            nested["nested1"] = "nested1value";
+            nested["nested2"] = "nested2value";
+
+            Assert.AreEqual("LEXICON of 3 items:\n[\"key1\"]= value1\n[\"key2\"]= 1\n[\"key3\"]=" +
+                " LEXICON of 2 items:\n  [\"nested1\"]= nested1value\n  [\"nested2\"]= nested2value", Serialize(lex));
```

Comment:
I'm really not in favor of the serialization file coming out different depending on what OS you produced it on, and being only able to read in files made by the same OS.  It's going to eventually be part of the save state of a game when it is used for comms (at least that's our plan), and as such should be copyable between different users on different machines.  ("Here's my bug report, with a copy of my .craft and my saved game folder....") About the only text reading or JSON reading tool out there that can't handle the files either way around is Windows' crappy Notepad.  Everything else can deal with text files written by "foreign" OS'es.